src/main/java/com/direwolf20/buildinggadgets/common/building/placement/package-info.java

```diff
@@ -0,0 +1,7 @@
+@ParametersAreNonnullByDefault
```

Comment:
But they are very handy...
Because all the methods in that package should only return nullable if specified - this helps a lot with idea's inspections.

Reply:
I’ve just disabled that specific inspection as it’s pointless

Reply:
I've got it active because I think it's useful.